mirage/config.ts

```diff
@@ -242,10 +242,6 @@ export default function(this: Server) {
         relatedModelName: 'registration',
     });
     this.get('/providers/registrations/:parentID/registrations/', getProviderRegistrations);
-    osfNestedResource(this, 'registration-provider', 'actions', {
```

Comment:
any reason we are removing this? We don't use it now, but this is still a valid endpoint in the API.

Reply:
I think having unused code adds clutter and potential confusion. This is quite easy to add when it's needed.

Reply:
Truth. Then out it goes!